domain_config/README.md

```diff
@@ -1,28 +1,38 @@
 Domain Config
 =============
 
-This module provides a per-domain configuration option so that you can change
-settings like the site name for each domain.
+This module provides a per-domain configuration option so that, for each
+domain, you can change configuration items of any type, such as settings like
+the site name.
 
-By default, these settings are also language-aware. An override may exist per
+By default, these overrides are also language-aware. An override may exist per
 domain, or per-language per domain, based on the prefixes used in the config
 file.
 
 Usage
 =====
 
 Currently, there is no user interface for changing settings, but you can load
-in new files that contain settings configuration per domain.
+in new files that contain configuration per domain.
 
 The pattern for override files is
-`domain.config.DOMAIN_MACHINE_NAME.LANGCODE.setting`, with the backup for
-language-insensitive files as `domain.config.DOMAIN_MACHINE_NAME.setting`.
+`domain.config.DOMAIN_MACHINE_NAME.LANGCODE.item.name`, with the backup for
+language-insensitive files as `domain.config.DOMAIN_MACHINE_NAME.item.name`,
+where `item.name` is the name of the configuration item being overridden.
 
-To override the site name, for instance, you have a file like the following:
+To override the site name, for instance, you begin by exporting the
+corresponding configuration item at the Configuration Synchronization screen
+for a single export:
+`admin/config/development/configuration/single/export`
+
+In this case, the type of configuration to be exported is "Simple
+configuration" and the particular item is named `system.site`.
+
+You should have a file like the following:
 
 ```YAML
 uuid: 536db351-7aec-407e-a746-70ba9cc9f190
```

Comment:
In nedjo's defence, a bit further it does state "An override file should contain only the specific data that you want to override."

Best to be very explicit about the need to exclude the UUID line.